### ISSUE TITLE
Not clear enough that the hidden-content component reveals something

### ISSUE THREAD

**Issue by muuvmuuv** [muuvmuuv/portfolio#18]
This component should be better highlighted as clickable. Maybe overlay a color and the text instead of a tooltip. This would also improve mobile accessibility!
